src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -2774,6 +2774,68 @@ public static void DCS_TypeWithDelegate()
         Assert.Equal(value.IntProperty, actual.IntProperty);
     }
 
+    [Fact]
+    public static void DCS_ResolveNameReturnsEmptyNamespace()
+    {
+        EmptyNsContainer instance = new EmptyNsContainer(new EmptyNSAddress());
+        var settings = new DataContractSerializerSettings() { MaxItemsInObjectGraph = int.MaxValue, IgnoreExtensionDataObject = false, PreserveObjectReferences = false };
+        EmptyNsContainer result = SerializeAndDeserialize(instance, null, settings, null, true);
+        bool flag = result.address != null;
+        if (flag)
+        {
+            throw new Exception("Address not null as per CSDMain:73231");
```

Comment:
We'd prefer to use `Assert.True(result.address != null, "Address not null as per CSDMain:73231")` instead of throwing exception. Besides, please remove bug info for CSDMain.